scripts/rebuildstaging-util

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+function usage() {
+    cat << EOF
+usage: $0 <staging-config-file>.yaml [-h|--help] [-v|--verbose] [--deploy|--no-push] [--skip-fetch]
+
+rebuild staging from yaml configuration
+
+  For ICDS-Staging env = scripts/icds-staging.yaml
+  For Staging env = scripts/staging.yaml
+
+    -h --help       print this help text
+    -v --verbose    display debugging output
+    --deploy        deploy after rebuild is complete
+    --no-push       do not push changes (cannot be used with --deploy)
+    --skip-fetch    assume local copy is already update date with remote
+    --enterprise    rebuild the enterprise branch
+EOF
+}
+
+filename=$1
+if [[ '' = $filename ]]
+then
+    echo "No config file specified."
+    exit 1
+fi
+shift
+while [[ $# > 0 ]]
+do
+    key="$1"
+    shift
+
+    case $key in
+      --deploy)
+        deploy=y
+        ;;
+      -h|--help)
+        usage
+        exit
+        ;;
+      -v|--verbose)
+        verbose=y
+        ;;
+      --skip-fetch)
+        skip_fetch=y
+        echo skip-fetch
+        ;;
+      --no-push)
+        no_push=y
+        echo no-push
+        ;;
+      --enterprise)
+        enterprise=y
+        echo enterprise
+        ;;
+      *)
+        usage
+        exit 1
+        ;;
+    esac
+done
+
+function rebuildstaging() {
+    echo "rebuilding staging branch, this might take a while..."
+    python scripts/rebuildstaging.py "$filename" "$@"
+}
+
+function get_deploy_env() {
```

Comment:
what about getting this from the config file:
```
## staging.yaml
deploy_env: staging

deploy_env=${grep deploy_env $filename | sed 's/deploy_env:\s//'}

Reply:
This would be much better 👍 
Thanks for the suggestion.